using_images/s2i_images/python.adoc

```diff
@@ -102,6 +102,10 @@ during the build. Only affects Django projects.
 |`*DISABLE_MIGRATE*`
 | Set it to a nonempty value to inhibit the execution of `manage.py migrate`
 when the produced image is run. Only affects Django projects.
+
+|`*PIP_INDEX_URL*`
+| Set it to a nonempty value to inhibit the execution of `manage.py migrate`
+when the produced image is run. Only affects Django projects.
```

Comment:
This looks like the same description as `DISABLE_MIGRATE` above it. Shouldn't it be something closer to the description from https://github.com/sclorg/s2i-python-container/pull/124/files, e.g.:

```
Set it to use a custom index URL or mirror to download the required packages during build process. This only affects packages listed in the *_requirements.txt_* file.
```